app/views/core/RootView.coffee

```diff
@@ -104,9 +104,16 @@ module.exports = class RootView extends CocoView
   switchToStudentMode: ->
     text = 'Switching to test student account..'
     noty({ text, type: 'success', timeout: 5000, killer: true })
```

Comment:
this should appear when we are actually spying not when we show modal. It should also appear when we click continue from modal